dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/inbound/NettyTCPReadRequestContext.java

```diff
@@ -145,182 +156,128 @@ public long read(long numBytes, int timeout) throws IOException {
 
         try {
             // Wait for the read operation to complete or timeout
-            return readFuture.get(60000, TimeUnit.MILLISECONDS);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(this, tc, "Waiting on read for channel: " + nettyChannel);
+            }
+            if (timeout == NO_TIMEOUT)
+                return readFuture.get();
+            else if (timeout == USE_CHANNEL_TIMEOUT)
+                return readFuture.get(60000, TimeUnit.MILLISECONDS);
```

Comment:
This needs to be updated so we use the inactivity timeout configured on the actual endpoint